cli/sawtooth_cli/submit.py

```diff
@@ -43,7 +43,14 @@ def do_submit(args):
     start = time.time()
 
     for batch_list in _split_batch_list(args, batches):
-        rest_client.send_batches(batch_list)
+        response = rest_client.send_batches(batch_list, args.wait)
+
+        # If told to wait, but timed out, print out statuses of batches
```

Comment:
If timed out the `POST /batch` route sends back `data` with batch statuses, but otherwise does not. So, as written it prints out a list of batches, along with their commit status (like committed or pending), in the case of a timeout. The comment could probably be more clear about how this works.

That said, Shawn just requested some fairly significant functionality changes, so all of this is going to get refactored pretty dramatically anyway.